pyseg/viewers/vesicle_graphs_fils_viewer.py

```diff
@@ -0,0 +1,68 @@
+# **************************************************************************
+# *
+# * Authors:     Scipion Team
+# *
+# * Unidad de  Bioinformatica of Centro Nacional de Biotecnologia , CSIC
+# *
+# * This program is free software; you can redistribute it and/or modify
+# * it under the terms of the GNU General Public License as published by
+# * the Free Software Foundation; either version 2 of the License, or
+# * (at your option) any later version.
+# *
+# * This program is distributed in the hope that it will be useful,
+# * but WITHOUT ANY WARRANTY; without even the implied warranty of
+# * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# * GNU General Public License for more details.
+# *
+# * You should have received a copy of the GNU General Public License
+# * along with this program; if not, write to the Free Software
+# * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA
+# * 02111-1307  USA
+# *
+# *  All comments concerning this program package may be sent to the
+# *  e-mail address 'scipion@cnb.csic.es'
+# *
+# **************************************************************************
+import glob
+from os.path import join
+
+from pyworkflow.gui.dialog import ToolbarListDialog
+from pyworkflow.utils.path import removeBaseExt
+from tomo3D.viewers.viewer_vtk import VtkPlot
```

Comment:
This forces this plugin to have tomo3D installed.

Probably we can capture the import error and not showing the viewers?

Reply:
I can see, there is a formal dependency in requirements. It's fine for me.